### ISSUE TITLE
Translations

### ISSUE THREAD

**Issue by HugoGiraudel** [HugoGiraudel/a11y.js#1]
I need to figure out how to deal with translations:

* get rid off them?
* like in a11y.css, in the data map?
* like Symfony, with YML/JSON/XLIFF translation files?
* is there a way with Chrome extensions to deal with this easily?

**Comment by ffoodd** [comment]
I just thought about it but since we're gonna play in the browser with JavaScript, may we check the default browser language?

That wasn't a good idea in a11y.css since we could only check the page's language (that may not be  the user's language) but with JavaScript (or at least in a browser extension) we should be able to check the browser's default language.

(but let user choose language is still needed !)

**Comment by HugoGiraudel** [comment]
> I just thought about it but since we're gonna play in the browser with JavaScript, may we check the browser's default language?

Smart.
